recipes/libnetcdf/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+export CFLAGS="-I$PREFIX/include $CFLAGS"
+export LDFLAGS="-L$PREFIX/lib $LDFLAGS"
+
+# https://www.unidata.ucar.edu/support/help/MailArchives/netcdf/msg11423.html
+export LD_LIBRARY_PATH=$PREFIX/lib:$LD_LIBRARY_PATH
```

Comment:
And if this is only needed for make check, maybe it should be 
`LD_LIBRARY_PATH=$PREFIX/lib:$LD_LIBRARY_PATH make check` (set LD_LIBRARY_PATH just for make check)
I don't think it should be set during `make`